### PR TITLE
docs streamline Azure quickstart

### PR DESCRIPTION
**Pull request by dtzar**
* Added Azure link to install cli tools
* Improved kubectl flow to get .kube config file from cluster

### CODE REVIEW

File: src/quickstart/provider/azure-acs/boot.md [hunk 1]
```diff
@@ -167,27 +168,16 @@ The Kubernetes cluster will take a few minutes to complete provisioning and conf
...
+az acs kubernetes install-cli
 ```
 
-Point `kubectl` at the kubernetes configuration file by setting the `KUBECONFIG` environment value:
+Download the master kubernetes cluster configuration to the ~/.kube/config file by running the following command:
 
-```
-export KUBECONFIG=~/.kube/k8sanddeis.config
+```console
+az acs kubernetes get-credentials --resource-group=$RG_NAME --name=$SERVICE_NAME
```
Comment (slack) [inline src/quickstart/provider/azure-acs/boot.md]: @dtzar Ah, this is great! What version of the `az` CLI do you need for this command to work?

File: src/quickstart/install-cli-tools.md [hunk 2]
```diff
@@ -34,13 +34,13 @@ Install the latest `helm` cli for Linux or Mac OS X by following the
...
 
-There are many ways to boot and run Kubernetes. You may choose to get up and running in cloud environments or locally on
-your laptop.
+There are many ways to boot and run Kubernetes. You may choose to get up and running in cloud environments or locally on your laptop.
 
 Cloud-based options:
 
 * [Google Container Engine](provider/gke/boot.md): provides a managed Kubernetes environment, available with a few clicks.
 * [Amazon Web Services](provider/aws/boot.md): uses Kubernetes upstream `kube-up.sh` to boot a cluster on AWS EC2.
+* [Azure Container Service](provider/azure-acs/boot.md): provides a managed Kubernetes environment.
```
Comment (slack) [inline src/quickstart/install-cli-tools.md]: Great add.

**Comment by bacongobbler** [comment]
If you wouldn't mind amending your commit such that it follows the [commit style guide](https://deis.com/docs/workflow/contributing/submitting-a-pull-request/#commit-style), that would be awesome! That way it'll get picked up by `deisrel` when we generate the release changelog. Thanks!

File: src/quickstart/provider/azure-acs/boot.md (hunk 1 above)
Comment (dtzar) [inline src/quickstart/provider/azure-acs/boot.md]: v0.1.0b11

**Comment by dtzar** [comment]
@bacongobbler fixed the commit style, sorry about that.  Please let me know if that works.

**Comment by bacongobbler** [comment]
thanks!

**Pull request merged**